Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -856,7 +865,28 @@ public extension String {
 			return self + padding[padding.startIndex..<padding.index(padding.startIndex, offsetBy: padLength)]
 		}
 	}
-	
+    
+    /// SwifterSwift: Deletes given prefix from string.
+    ///
+    ///   "Hello, World!".deletingPrefix("Hello, ") -> "World!"
+    ///
+    /// - Parameter prefix: Prefix to delete from string.
+    /// - Returns: The string after prefix deleting.
+    public func deletingPrefix(_ prefix: String) -> String {
```

Comment:
I like these extensions 👏 Do you think we should name it `removingPrefix`?

Reply:
Yes, maybe "removing" is better word than "deleting". I checked there's "remove" word used more than "delete" in code base. Should I change methods names to removingPrefix and removingSuffix ?

Reply:
@n0an Sounds good to me 👌 Please add CHANGELOG entries as well